docs/develop/feather-js/terra-classic.mdx

```diff
@@ -0,0 +1,9 @@
+import Admonition from '@theme/Admonition';
+
+# Using Terra Classic
+
+<Admonition typpe='caution'>
```

Comment:
@ericHgorski ^^ small typo in the word "typpe"

Reply:
```suggestion
<Admonition type='caution'>
```